src/components/index.d.ts

```diff
@@ -611,6 +608,19 @@ export declare const NumberInput: {
   }
 }
 
+export declare const Faq: {
+  ({ questions, open }: { questions: any; open: any }): JSX.Element
```

Comment:
this used to be typed with `: boolean` but that was throwing an error `"JSX element type 'boolean' is not a constructor function for JSX elements."` and I noticed I've been having success with react component exports being of type `JSX.Element` which I believe is correct.